stdlib/public/core/SmallString.swift

```diff
@@ -442,12 +442,32 @@ internal func _bytesToUInt64(
   // FIXME: This should be unified with _loadPartialUnalignedUInt64LE.
   // Unfortunately that causes regressions in literal concatenation tests. (Some
   // owned to guaranteed specializations don't get inlined.)
+  guard _fastPath(c != 8) else {
+    return unsafe input.withMemoryRebound(to: UInt64.self, capacity: 1) { unsafe $0.pointee.littleEndian }
```

Comment:
```suggestion
    return unsafe UnsafeRawPointer(input).loadUnaligned(as: UInt64.self).littleEndian
```
Technically the `pointee` property should be called only on pointers well-aligned for the `Pointee` type, and the context doesn't provide that guarantee. The same applies to the similar load operations at lines 455 and 462.